networking_cisco/tests/unit/cisco/l3/test_l3_router_appliance_plugin.py

```diff
@@ -551,6 +552,22 @@ def test_router_update_gateway_to_empty_with_existed_floatingip(self):
             'test_router_update_gateway_to_empty_with_existed_floatingip', 1,
             1)
 
+    def test_add_namespace_binding(self):
+        r = None
+        with self.router() as router:
+            r = router['router']
+        adm_context = n_context.get_admin_context()
+        query = adm_context.session.query(
+            l3_models.RouterHostingDeviceBinding)
+        hdb = query.first()
```

Comment:
let's enhance the test a bit like to use router_id to make sure its indeed the one we got.